lib/screens/home_page.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:app/screens/profile_page.dart';
+import 'package:app/screens/teams_page.dart';
+import 'package:flutter/material.dart';
+
+class HomePage extends StatefulWidget {
+  const HomePage({super.key});
+
+  @override
+  State<HomePage> createState() => _HomePage();
+}
+
+class _HomePage extends State<HomePage> {
+  // List of page screens
+  final List<Widget> _widgetOptions = <Widget>[
+    const ProfilePage(),
+    const TeamPage(),
+  ];
+  int _selectedIndex = 0;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      // Set the page based on selected index
+      body: _widgetOptions.elementAt(_selectedIndex),
```

Comment:
It is more common to write `_widgetOptions[_selectedIndex]` instead of `_widgetOptions.elementAt(_selectedIndex)`.